CMakeLists.txt

```diff
@@ -1870,6 +1870,10 @@ if(ENABLE_PLUGIN_CGROUP_NETWORK)
 endif()
 
 if(ENABLE_PLUGIN_SYSTEMD_JOURNAL)
+        if(NOT SYSTEMD_FOUND)
```

Comment:
Hello @Ferroin ,

Your PR is showing the message on systems without `systemd`, but it is also stopping the whole compilation with messages:

```sh
CMake Error at CMakeLists.txt:1874 (message):
  Systemd journal plugin requires systemd, but systemd was not found.


gmake: Makefile: No such file or directory
gmake: *** No rule to make target 'Makefile'.  Stop.
```

and 

```sh
-- Configuring incomplete, errors occurred!
```

So, I have to disable plugin manually to proceed. I supposed this is the goal, please, can you confirm?

Another doubt, are we going to do this with other collectors like `IMPI` and `XEN` ?

Reply:
> So, I have to disable plugin manually to proceed. I supposed this is the goal, please, can you confirm?

When building by hand, yes. When using the installer this should be auto-detected. The idea here is to give a clear failure whenever possible instead of silently disabling things, because that makes it easier for anybody building by hand to figure out _why_ some feature is not being built.